spec/support/factory_girl.rb

```diff
@@ -6,6 +6,7 @@
     if ENV['OSEM_FACTORY_LINT'] != 'false'
       mock_commercial_request
       FactoryGirl.lint
+      DatabaseCleaner.clean_with(:truncation)
```

Comment:
Just realised, we can not truncate the database here because this will delete the seeds, or we need to load the seeds after truncating.